gitversion/entrypoint.sh

```diff
@@ -2,12 +2,21 @@
 set -o errexit   # abort on nonzero exitstatus
 set -o pipefail  # don't hide errors within pipes
 
-GIT=/git
-GEN=/gen
+if [ ! -z "$GIT_PATH" ]; then
```

Comment:
it would be great if the variable GIT_PATH and GEN_PATH were mentioned in the documentation.

Reply:
Fixed.